components/experimental/src/units/ratio.rs

```diff
@@ -21,15 +21,28 @@ use super::provider::{Base, SiPrefix};
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
 pub struct IcuRatio(Ratio<BigInt>);
 
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
 pub enum IcuRatioError {
```

Comment:
issue: we decided to not have catch-all error types going forward. Please make a custom error type for each function that can fail, which only contains error variants that are applicable to this function.

Reply:
do you mean that making an error depending on the functionality:

For example: `InvalidRatioString` or `ErrorInFromStr` ?

Reply:
Yes, `RatioFromStrError`

Reply:
done

Reply:
I meant a different type `pub enum RatioFromStrError { DivisionByZero, MultipleSlashes, ... }`

Reply:
OMG, that is a reverse change. 

let me try to do it

Reply:
done